src/verify.c

```diff
@@ -719,6 +719,22 @@ int verify_main(int argc, char **argv)
 		}
 	}
 
+	/* If --fix was specified but --force and --picky are not present, check
+	 * that the user is only fixing to their current version. If they are
+	 * fixing to a different version, print an error message that they need to
+	 * specify --force or --picky. */
+	if (cmdline_option_fix && !is_current_version(version)) {
```

Comment:
Do you want to add special handling for `verify --install` as well?

Reply:
yes.

Reply:
Actually, per our meatspace convo, the --install option should be fixed up outside this PR, and should not require --force or --picky to be run, and possible made into its own sub-command.

Reply:
Agreed. Let's not special-case --install in this case for now.